release.yaml

```diff
@@ -56,3 +56,7 @@ sriov:
 - 101.0.2+up0.1.0
 sriov-crd:
 - 101.0.2+up0.1.0
+rancher-gke-operator:
+- 101.0.0+up1.1.5
```

Comment:
`rancher-gke-operator` and `rancher-gke-operator-crd` were released with version `101.0.0+up1.1.5` for 2.7.0 (see https://github.com/rancher/charts/tree/release-v2.7/charts/rancher-gke-operator) so I don't think we can update this chart any longer.
We need to introduce a new one (I guess 101.0.1+up1.1.5) with these constraints updated.

Reply:
Ahh that makes sense. I would think `101.1.0` since introducing 1.25 support is pulling in new functionality, not a bug fix but it's also a small change code wise. Thoughts?